src/routes/web.php

```diff
@@ -30,3 +30,5 @@
 
 //Posts Routes
 Route::post('/post', [PostController::class, 'create']);
+//Delete Posts
```

Comment:
I noticed you're not using middlewares on these routes (Post and Delete) does this mean that even unauthenticated users can post and delete posts?

Reply:
I added the post and delete route to the middleware group as the user needs to be authenticated to be able to create a post and delete.